src/main/java/fr/xephi/authme/message/MessageKey.java

```diff
@@ -146,6 +146,9 @@
     /** The server is full, try again later! */
     KICK_FULL_SERVER("on_join_validation.kick_full_server"),
 
+    /** An error occurred: unresolved player hostname! **/
+    KICK_UNRESOLVED_HOSTNAME("error.kick_unresolved_hostname"),
```

Comment:
Please run the `verifyMessages` tool task so that all YML files will be updated with this new message :smiley:

Reply:
uh the tool seems to be broken, it will place "null" instead of the missing translation values

Reply:
Fixed on master, thanks for the hint :smile: